pkgs/tools/networking/connman/connman.nix

```diff
@@ -125,15 +120,15 @@ stdenv.mkDerivation rec {
     ++ optionals (!enableClient) [ "--disable-client" ]
     ++ optionals (!enableDatafiles) [ "--disable-datafiles" ]
     ++ optionals (enableOpenconnect) [
-      "--enable-openconnect=builtin"
```

Comment:
What's the reason for switching from builtin to plugin?

Reply:
I haven't found any mention of `=builtin` in the output of `./configure --help`. Hence I figured these configure flags got ignored. I guess this still hasn't bothered the necessary binaries to be used since (e.g) `--with-openconnect=${openconnect}/bin/openconnect` was used as well.

Reply:
See `configure.ac` and `Makefile.plugins`, grep for `OPENCONNECT_BUILTIN`. Without `builtin` these extensions are built a plugins.

Reply:
:+1: .